### ISSUE TITLE
Save draft and publish return red triangle errors (unable to create a record)

### ISSUE THREAD

**Issue by carsonicator** [inveniosoftware/invenio-app-rdm#314]
Package version (if known): invenio-cli, version 0.17.2

## Describe the bug
I'm unable to create a record. Publishing a record returns me to the record creation page with error triangles in all section headers, including those that are "COMING SOON".

## Steps to Reproduce
1. After installation, create a user account and log in.
2. Click "Uploads" in top menu bar.
3. Click "New Deposit".
4. Enter all required information and optional info if you like.
5. Click on "Publish".
6. Click "Publish" on the popup.
7. See red triangles as you are returned to the record page, but no error message.

## Expected behavior

Record is created as long as all required information is entered. I would expect an error message if these criteria weren't met.

## Screenshots (if applicable)

<img width="1188" alt="Screen Shot 2020-10-14 at 8 33 33 AM" src="https://user-images.githubusercontent.com/11506832/96004369-edf55800-0e00-11eb-8134-14194cb5aa2d.png">

## Additional context
I had a similar issue with the August release ( #272 ).

**Comment by ppanero** [comment]
@carsonicator sorry for the dealy, was off yesterday. One last quick question cuz it was not clear to me, and I cannot reproduce the error, are you accessing the browser also via `127.0.0.1` and *not* localhost?

**Comment by carsonicator** [comment]
@ppanero yes, I'm using `127.0.0.1`. The full URL of the record I created is `https://127.0.0.1/uploads/nt41t-pds75`. If I modify the record and try to save a draft, an error like my previous screenshot appears. Interestingly, unlike when I initially created the record and received an error, my attempts to modify the existing record are not saved if I reload the page. So it appears that I can create a record, receive the red triangle errors, reload the page in the browser, click 'uploads', and the record appears despite the previous error. However, if I edit an existing record (I added Sara Gonzales as an additional creator in this example) and save the draft, I get the same red triangle errors but when I reload the page, click 'Uploads', and look at the record I modified, the changes have not been saved. Does this make sense?
